account_invoice_export/models/account_move.py

```diff
@@ -65,12 +65,13 @@ def _export_invoice(self):
         file_data = self._get_file_for_transmission_method()
         headers = self.transmit_method_id.get_transmission_http_header()
         url = self.transmit_method_id.get_transmission_url()
-        res = requests.post(url, headers=headers, files=file_data)
+        res = requests.post(url, headers=headers, files=file_data, timeout=5)
```

Comment:
Isn't 5 seconds a bit low? Looking into odoo code it's generally between 15 and 30s. Could be good to add a parameter (a TODO in the code is also fine while working on the mig)

Reply:
I changed it to 10.
And also left a comment to improve with configuaration.